src/oas2/transformers/params.ts

```diff
@@ -19,7 +20,7 @@ import {
 } from 'swagger-schema-official';
 
 function chooseQueryParameterStyle(
```

Comment:
This function is not taking a `FormDataParameter` as well — so the name is probably not ok anymore; in any case this is only used in the `QueryParameter` path so I'm not too sure why this change is here

Reply:
Oh, overshight! Removing!